docs/help/account/user-account-settings.md

```diff
@@ -0,0 +1,28 @@
+# User account settings
+
+Accessible from the Dashboard, the Account Settings page allows you to quickly modify certain elements related to your account.
+
+From this page, you can modify:
+
+- your first and last name
+- your password
+- your username
+
+## Modify Username
+
+Your username serves also as your slug, which is part of the URL of your documentation and hubs except if you use custom domains.
+Modify your username will impact your URLs
```

Comment:
We should explain how it will impact the URLS, and that Bump.sh doesn't automatically redirect after a change. Also, I think we should write "modifying", and a final point is missing here.

Reply:
👍